stories/topics/proc-azure-cancel-deployment.adoc

```diff
@@ -0,0 +1,33 @@
+[id="azure-cancel-deployment_{context}"]
+
+= Canceling {AAPonAzureName} deployments
+
+[role="_abstract"]
+You can gracefully cancel an {AAPonAzureName} deployment. 
+
+.Procedure
+
+. Login to the deployment engine to display the progress of the deployment steps in the *Ansible Automation Platform Deployment Engine* page.
+Refer to xref:azure-monitor-deployment-engine_aap-azure-install[Monitoring deployments on the {PlatformNameShort} Deployment Engine] for information on accessing and logging into the *Ansible Automation Platform Deployment Engine* page.
+
+. To cancel the deployment, click btn:[Cancel Deployment] and confirm.
++
+This action cancels all the remaining steps in the deployment, including the currently running step. It also cancels pending steps in the deployment.
++
+The status for the steps that have not been executed updates to *Cancelled*.
+To view the deployment processes on Azure, open the *Overview* page for your deployment on the Azure web interface and select *Deployments*.
++
+[IMPORTANT]
+====
+Cancelling the deployment does not delete the managed application from your Azure subscription.
+To avoid incurring costs for the managed application and other resources that are still running, you must delete them.
+====
+. To delete Azure resources, navigate to the resource group for your deployment in the Azure portal. Select the resources you want to delete and click btn:[Delete].
+For more information about deleting resources, refer to link:https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resources-portal[Manage Azure resources by using the Azure portal] in the Microsoft Azure documentation.
+
+You cannot restart a deployment after you have canceled it.
```

Comment:
What does "restart a deployment" here mean? After customers cancel a deployment and delete all the cancelled deployment related resources from Azure, they can kick off a new deployment if they want.

Reply:
I deleted the sentence. It was mentioned in the demo video.